pkg/rpmmd/repository.go

```diff
@@ -160,6 +160,7 @@ type PackageSpec struct {
 	Version        string `json:"version,omitempty"`
 	Release        string `json:"release,omitempty"`
 	Arch           string `json:"arch,omitempty"`
+	NEVRA          string `json:"nevra,omitempty"`
```

Comment:
I wasn't (much) involved with osbuild https://github.com/osbuild/osbuild/pull/1933 - why was this new string added? It seems it contains the concatenated information that is already provided by "name,epoch,version,arch,release", why do we need it again?